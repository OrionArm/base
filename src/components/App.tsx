import React, { Component } from "react"
import ButtonAppBar from "./ButtonAppBar"

type Props = {
  head?: any
  content?: any
}

class App extends Component<Props> {
  render() {
    return (
      <div className="profile">
        <ButtonAppBar />
        <div className="content">Content</div>
      </div>
    )
  }
}

export default App
