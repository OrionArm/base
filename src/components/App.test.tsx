import React from "react"
import { shallow } from "enzyme"
import App from "components/App"

describe("App", () => {
  const render = shallow(<App />)
  test("Render UI", () => {
    expect(render).toMatchSnapshot()
  })
})
