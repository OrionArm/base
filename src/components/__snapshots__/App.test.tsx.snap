// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Render UI 1`] = `
<div
  className="profile"
>
  <WithStyles(ButtonAppBar) />
  <div
    className="content"
  >
    Content
  </div>
</div>
`;
